ntrans_combine.py

```diff
@@ -0,0 +1,115 @@
+"""
+Reads the .csv files containing N-grams that were generated in ntrans_dataprep.py
+
+Inserts every N-gram into a collections.Counter, to determine the 10K most frequent N-grams of each sorted
+(2-grams, 3-grams, etc.).
+
+These N-grams are thereafter written to separate .csv files.
+"""
+
+import csv
+import collections
+import pathlib
+from typing import Dict, List, Tuple, Union
+
+
+NGramCounter = Tuple[str, int]
+
+
+def delete_chunkfiles():
+    """
+    Deletes the generated chunked csv files after the finalized csv files have been finalized.
+    """
+    confirm = (
+        str(
+            input(
+                "Are you sure you want to delete the chunked data files? This can not be undone. (y/n) "
+            )
+        )
+        .lower()
+        .strip()
+    )
```

Comment:
I don't like how black formats this. Does it do a better job if the long string is in a separate variable?

```suggestion
    prompt = "Are you sure you want to delete the chunked data files? This can not be undone. (y/n) "
    confirm = (
        str(
            input(
                prompt
            )
        )
        .lower()
        .strip()
    )
```

That might collapse a lot when you black